media/js/firefox/sync.js

```diff
@@ -32,15 +33,15 @@
     };
 
     // Variations 1-5 are Firefox
-    if (window.isFirefox()) {
+    if (client.isFirefox) {
```

Comment:
Looks like there's nothing for Firefox for iOS to fall into here below, should we add `!client.isFirefoxiOS`?